src/Components/CreateImageWizard/formComponents/Packages.js

```diff
@@ -0,0 +1,84 @@
+import React, { useState, useRef, useEffect } from 'react';
+import useFormApi from '@data-driven-forms/react-form-renderer/use-form-api';
+import useFieldApi from '@data-driven-forms/react-form-renderer/use-field-api';
+import { DualListSelector, Button, TextContent } from '@patternfly/react-core';
+import api from '../../../api';
+import PropTypes from 'prop-types';
+import '../WizardStepPackages.scss';
```

Comment:
Please remove this import since the file no longer exists.

Reply:
Fixed by @Gundersanne in https://github.com/Gundersanne/image-builder-frontend/commit/16413280a3c41ed778fcd0d91c655b0e326d0adc